.github/workflows/build-release.yml

```diff
@@ -0,0 +1,250 @@
+name: Build release
+
+on:
+  workflow_dispatch:    # Allow manual triggers
+    inputs:
+      tag:
+        description: 'Release tag (format: major.minor.patch)'
+        required: true
+      draft:
+        description: 'Create a draft (unpublished) release (default: false)'
+        type: boolean
+        default: false
+      prerelease:
+        description: 'Identify the release as a prerelease (default: false)'
```

Comment:
I guess `draft` in draft release is similar with `draft` in draft PR.
For pre-release, I found https://semver.org/#spec-item-9.
Is pre-release for something like alpha version or beta version?

Reply:
These features are related to GitHub Release system.

When you release a new version in GitHub System, you can use `draft` and `prerelease` items.

Pre-release in GitHub Release means that the release is not ready for production and may be unstable.
Draft in GitHub Release means that if you're ready to publicize your release but to work on the release later, use `draft` release.